### PR TITLE
docs(SDB): add planned maintenance doc MTA-4837

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by ldecarvalho-doc — APPROVED** [review]

File: serverless/sql-databases/reference-content/planned-maintenance.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+dates:
+  validation: 2024-07-09
+  posted: 2024-07-09
+---
+
+## Overview
+
+To add features, deploy security fixes or corrective patches, we perform regular maintenance operations. **Most maintenance operations are transparent to you**, and have no impact on your database availability, or performance.
+
+However, **some maintenance operations may impact your database availability** or require actions from your side. In this case, **we will inform you beforehand**, so you can **follow the recommendations below** to ensure the impact on your application is minimal.
```
Comment (nerda-codes) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:18]:
```suggestion
However, **some maintenance operations may impact your database availability** or require actions on your side. In this case, **we will inform you beforehand**, so you can **follow the recommendations below** to ensure that the impact on your application is minimal.
```

Comment (nerda-codes) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:18]:
```suggestion
However, **some maintenance operations may impact your database availability** or require actions on your side. In this case, **we will inform you beforehand**, so you can **follow the recommendations below** to ensure that the impact on your application is minimal.
```

File: serverless/sql-databases/reference-content/planned-maintenance.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The maintenance operations particularly concern our [connection pooler](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) for now, as we are working to make it as transparent as possible in future Serverless SQL Databases updates.
```
Comment (nerda-codes) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:20]:
```suggestion
The maintenance operations particularly concern our [connection pooler](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) for now, as we are working on making it as transparent as possible in future Serverless SQL Databases updates.
```

File: serverless/sql-databases/reference-content/planned-maintenance.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+- Require your **SQL client to reconnect afterward**
+
+To avoid requiring any manual actions on your side, **we recommend you ensure that your applications or SQL clients can reconnect automatically**. This is the default behavior with most web application frameworks or software relying on SQL Databases.
+
+<Message type="note">
+When the maintenance operation starts, database queries pending (such as transactions being processed) will still be processed before SQL clients are disconnected, as long as transactions do not last more than 1 minute.
+</Message>
+
+## Maintenance period & communication delay
```
Comment (nerda-codes) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:36]:
```suggestion
## Maintenance period and communication delay
```

File: serverless/sql-databases/reference-content/planned-maintenance.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+We will **communicate the maintenance period at least 1 week in advance** for low-impact operations, such as unavailability below 1 minute, and **up to 1 month in advance** for higher-impact operations.
+
+By default, we plan a maintenance period **every Tuesday between 14:30 am and 14:45 am (CEST)**, that disconnects SQL clients from your database **for at most 10 seconds**.
```
Comment (nerda-codes) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:40]:
```suggestion
By default, we plan a maintenance period **every Tuesday between 14:30 pm and 14:45 pm (CEST)**, that disconnects SQL clients from your database **for at most 10 seconds**.
```

File: serverless/sql-databases/reference-content/planned-maintenance.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+However, **some maintenance operations may impact your database availability** or require actions from your side. In this case, **we will inform you beforehand**, so you can **follow the recommendations below** to ensure the impact on your application is minimal.
+
+The maintenance operations particularly concern our [connection pooler](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) for now, as we are working to make it as transparent as possible in future Serverless SQL Databases updates.
+
+## Impact on your application and recommendations
+
+A maintenance operation impacting your SQL client connections to the database will usually:
+
+- Disconnect your SQL connections to the database **during 10 seconds**
```
Comment (fpagny) [inline serverless/sql-databases/reference-content/planned-maintenance.mdx:26]:
```suggestion
- Disconnect your SQL connections to the database **during less than 1 minute**
```
After technical discussion, even if the operation should take 10 seconds in most cases, mentioning 1 minute maximum delay is more cautious

**Review by fpagny — CHANGES_REQUESTED** [review]
Small change to perform on disconnection delay (based on technical discussions requiring update on the disconnection delay mentioned)

**Review by fpagny — APPROVED** [review]
Ok for me 👍

**Pull request merged**